Public/Src/Deployment/NugetPackages.dsc

```diff
@@ -122,9 +122,11 @@ namespace NugetPackages {
                 cacheHashing,
             ]),
             sdks,
-            osxX64,
+            ...addIf(!BuildXLSdk.Flags.genVSSolution,

```

Comment:
>addIf [](start = 15, length = 5)

seperate pr?

Reply:
Sure.

---
In reply to: [283919613](https://github.com/microsoft/BuildXL/pull/301#discussion_r283919613) [](ancestors = 283919613)